background/state.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  version: null
```

Comment:
It seems to work, but wouldn't it be future-proof/extensible to have something like

```javascript
class State {
  version = null;
}

const state = new State();

export default state;
```